jaxrs/src/main/java/org/killbill/billing/jaxrs/json/PaymentAttemptJson.java

```diff
@@ -31,7 +31,7 @@
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 
-@ApiModel(description = "Payment attempt")
```

Comment:
Should we actually populate the `description` field everywhere? While for things like `Account`, it would not provide much info, I think it would be useful for esoteric models like `Combo*`.

Doesn't need to happen in this PR, we could start opening tickets for these things.

Reply:
Yeah, i think at this point my swagger changes are mostly targeted around non descriptive things -- we had somehow abandoned to build a solid api doc around swagger, moving to manual Slate. By i agree one does not prevent the other. 

I am on board to iterative passes on swagger annotation to make it as good as possible -- inclu descriptions.

Reply:
👍Slate for the bulk of the docs, but ensure that short descriptions in Swagger are meaningful nonetheless.

For instance, it was reported yesterday that in Swagger, both of the APIs look the same, which is confusing:

* https://github.com/killbill/killbill/blob/eba600f471a26a5ef875b35601ceeb9cbf65a4aa/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/PaymentResource.java#L380
* https://github.com/killbill/killbill/blob/eba600f471a26a5ef875b35601ceeb9cbf65a4aa/jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/PaymentResource.java#L406